build/run

```diff
@@ -118,16 +110,10 @@ if [[ -n "${CACHE_DIR}" ]]; then
         -v ${CACHE_DIR}:${BUILDER_HOME}/go/src/${BUILD_REPO}/.cache"
 fi
 
-# we copy credential files for github access
-[[ -f ${HOME}/.netrc ]] && NETRC_ARGS="-v ${HOME}/.netrc:${BUILDER_HOME}/.netrc"
-
 tty -s && TTY_ARGS=-ti || TTY_ARGS=
 
-# docker seems more reliable when running with net=host.
-if [ -z "${DISABLE_HOST_NETWORK}" ]; then
-    NET_ARGS="--net=host"
-fi
-
+DISABLE_NESTED_DOCKER=""
```

Comment:
Why is this declared empty?

Reply:
initially, I removed this with other var as I was getting `unbound var` in this too but later I was getting `/bin/bash: line 1: build: command not found and I tried to understand from where this error is coming and when I did like this it was good.

Reply:
https://github.com/subhamkrai/rook/runs/3482498626?check_suite_focus=true#step:6:482 link of the error, I can share more links of the same if needed